core/indigo-core/molecule/src/sequence_saver.cpp

```diff
@@ -1048,3 +1051,161 @@ void SequenceSaver::saveIdt(KetDocument& doc, std::vector<std::deque<std::string
         seq_text += seq_string;
     }
 }
+
+static const char* get_helm_class(MonomerClass monomer_class)
+{
+    switch (monomer_class)
+    {
+    case MonomerClass::AminoAcid:
+        return kHELMPolymerTypePEPTIDE;
+    case MonomerClass::Base:
+    case MonomerClass::Sugar:
+    case MonomerClass::Phosphate:
+    case MonomerClass::RNA:
+    case MonomerClass::DNA:
+    case MonomerClass::Linker:
+    case MonomerClass::Terminator:
+        return kHELMPolymerTypeRNA;
+    case MonomerClass::Unknown:
+    case MonomerClass::CHEM:
+    default:
+        return kHELMPolymerTypeCHEM;
+        break;
+    }
+    return kHELMPolymerTypeCHEM;
+}
+
+std::string SequenceSaver::saveHELM(KetDocument& document, std::vector<std::deque<std::string>> sequences)
+{
+    std::string helm_string = "";
+    int peptide_idx = 0;
+    int rna_idx = 0;
+    int chem_idx = 0;
+    using MonomerInfo = std::tuple<HELMType, int, int>;
+    constexpr int polymer_type = 0;
+    constexpr int polymer_num = 1;
+    constexpr int monomer_num = 2;
```

Comment:
what is monomer_num, polymer_num?
is it minimal number of monomers or anything else

Reply:
Current polymer num and current simple monomer num.

Reply:
They need to process connection later

Reply:
Looks like it was used to address tuple, but then tuple was changed to pair.
I removed it.